apps/test/unit/code-studio/components/teacher-homepage/CourseCardTest.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import  CourseCard from '@cdo/apps/templates/teacherHomepage/CourseCard';
+
+const NONE_ENROLLED = {
```

Comment:
Nit:  Not sure we explicitly call this out anywhere, but I tend to use all caps just for simple constants (i.e. const CARD_HEIGHT = 12). I probably would do these in camel case (noneEnrolled), though it doesn't really matter.